packages/website/components/gateway.js

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react'
+
+export default function Gateway() {
+  const [cid, setCid] = useState(
+    'bafybeiedv7sowwxamly4oicivudp45rsfvbklnf3fvbvonxrwoxqylhtwq/0.json'
+  )
+  // QmcMBwDT8wEjNaVgFpQjhZMKPEgHAwNHTErpZTvPzLdvvi
+  const handleCIDLink = (e) => {
+    e.preventDefault()
+    window.open(`https://nftstorage.link/ipfs/${cid}`, '_blank')
+  }
+
+  return (
+    <div id="try-it" className="bg-white relative overflow-hidden">
+      <div className="mx-auto max-w-7xl w-full pt-16 pb-20 lg:py-32 px-6">
+        <div className="relative">
+          <h1 className="font-open-sans text-3xl font-bold text-black lg:text-4xl">
+            <span className="block">The NFT.Storage Gateway</span>
+          </h1>
+          <p className="mt-3 text-lg text-gray-900 sm:text-xl md:mt-5 md:max-w-4xl">
+            The NFT.Storage gateway is optimized for NFT content stored on IPFS,
+            and is especially fast if that content is stored on NFT.Storage.
+            Plug your NFT&apos;s content ID (CID) below to try it out.
+          </p>
+        </div>
+        <form>
+          <div className="sm:flex rounded-md max-w-3xl mt-12 space-y-2 sm:space-y-0">
+            <span className="sm:inline-flex items-center sm:px-3 sm:rounded-l-md sm:border-2 sm:border-r-0 sm:border-black sm:bg-gray-100 text-gray-500 md:text-lg">
+              https://nftstorage.link/ipfs/
+            </span>
+            <input
```

Comment:
Can we have this bigger in Desktop resolution? In mobile it won't be possible, but feels like we have more space and we could get the size to show the path for json?